config/sync/user.role.site_manager.yml

```diff
@@ -7,23 +7,74 @@ label: 'Site Manager'
 weight: -3
 is_admin: null
 permissions:
+  - 'access administration pages'
   - 'access content overview'
+  - 'access contextual links'
+  - 'access environment indicator'
   - 'access file_browser entity browser pages'
+  - 'access files overview'
```

Comment:
Do we really wanna give them this page? they will have access to the media overview page

Reply:
The site manager is a pretty high-level role. I think this should be fine but I can remove it and grant later. Better to give than to take awayeth.